src/main/drivers/pwm_output.c

```diff
@@ -48,6 +48,8 @@ static uint16_t freqBeep = 0;
 static bool pwmMotorsEnabled = false;
 static bool isDshot = false;
 
+#define CHAN_TO_HALCHAN(n) (((n) - 1) * 4)
```

Comment:
This is bad idea ... it will break a lot of things.
Either store `TIM_CHANNEL_x` in channel, or define function in timer.h that will return correct `TIM_CHANNEL_x` value from `timerHardware_t`